### PR TITLE
KAFKA-17717 Remove ConfigUtils#translateDeprecatedConfigs and tests

### PR DESCRIPTION
**Pull request by mingdaoy**
https://issues.apache.org/jira/browse/KAFKA-17717

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by mingdaoy** [comment]
Hi maintainers, could you please review and approve this workflow when you have a chance? Thanks!

**Comment by mingdaoy** [comment]
@brandboat Thank you!

**Review by frankvicky — COMMENTED** [review]
Hi @mingdaoy 
It seems that some build error occur.
Please build at local firstly then push it again.

**Review by chia7712 — APPROVED** [review]

**Pull request merged**
